crypto/s2n_tls13_keys.c

```diff
@@ -186,6 +186,7 @@ int s2n_tls13_derive_early_secrets(struct s2n_tls13_keys *keys, struct s2n_psk *
     } else {
         /* Sanity check that an early secret exists */
         ne_check(psk->early_secret.size, 0);
+        lte_check(psk->early_secret.size, keys->extract_secret.size);
```

Comment:
"lte" isn't sufficient here. It needs to actually be equal to. Otherwise, extract_secret could contain the early secret + ??? because its size will be set incorrectly.